internal/app_config.go

```diff
@@ -80,6 +80,15 @@ func (c AppConfig) Get(key string) string {
 	return ""
 }
 
+func (c AppConfig) GetOrDie(key string) string {
+	val := c.Get(key)
+	if len(val) == 0 {
+		fmt.Println("ERROR: Missing", key, "in sota.toml")
```

Comment:
`in sota.toml` is a bit confusing IMO, since the "get" code tries to find and get the key value across all configuration files.

Reply:
ah - good catch. that's copy/paste from the old stuff.

Reply:
https://github.com/foundriesio/fioconfig/pull/45/commits/71a1cea2b191a3f410892f8c758935699c4f2c69